From david-b@pacbell.net  Thu Aug 10 13:15:50 2006
From: David Brownell <david-b@pacbell.net>
To: linux-pm@lists.osdl.org
Subject: make suspend quieter
Date: Sun, 9 Jul 2006 16:28:28 -0700
Cc: Linus Torvalds <torvalds@osdl.org>, Greg KH <greg@kroah.com>, Pavel Machek <pavel@ucw.cz>
Message-Id: <200607091628.29269.david-b@pacbell.net>

Fix a goof in Linus' recent PM API updates:  don't emit any messages in the
typical NOP "already suspended it" late suspend case.

Signed-off-by: David Brownell <dbrownell@users.sourceforge.net>
Acked-by: Pavel Machek <pavel@ucw.cz>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/base/power/suspend.c |    5 -----
 1 file changed, 5 deletions(-)

--- gregkh-2.6.orig/drivers/base/power/suspend.c
+++ gregkh-2.6/drivers/base/power/suspend.c
@@ -102,11 +102,6 @@ static int suspend_device_late(struct de
 {
 	int error = 0;
 
-	if (dev->power.power_state.event) {
-		dev_dbg(dev, "PM: suspend_late %d-->%d\n",
-			dev->power.power_state.event, state.event);
-	}
-
 	if (dev->bus && dev->bus->suspend_late && !dev->power.power_state.event) {
 		dev_dbg(dev, "LATE %s%s\n",
 			suspend_verb(state.event),
